src/main/java/roomescape/controller/ReservationTimeController.java

```diff
@@ -0,0 +1,41 @@
+package roomescape.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.ReservationTime;
+import roomescape.dto.ReservationDto;
+import roomescape.dto.ReservationTimeDto;
+import roomescape.repository.ReservationTimeRepository;
+
+import java.net.URI;
+import java.util.List;
+
+@Controller
+@ResponseBody
+public class ReservationTimeController {
+
+    ReservationTimeRepository reservationTimeRepository;
+
+    public ReservationTimeController(ReservationTimeRepository reservationTimeRepository) {
+        this.reservationTimeRepository = reservationTimeRepository;
+    }
+
+    @PostMapping("/times")
+    public ResponseEntity<ReservationTimeDto> addTime(@RequestBody ReservationTimeDto dto) {
+        ReservationTimeDto time = reservationTimeRepository.addTime(dto);
+        return ResponseEntity.ok().body(time);
+    }
+
+    @GetMapping("/times")
+    public ResponseEntity<List<ReservationTime>> timeList() {
+        List<ReservationTime> list = reservationTimeRepository.findTimeList();
```

Comment:
자료형이 드러나는 네이밍을 사용할 경우 해당 자료형이 변경될 때 이름도 함께 변경될 수 있습니다!
다른 부분도 함께 확인해보세요!
```suggestion
        List<ReservationTime> list = reservationTimeRepository.findTimes();
```